arxiv/arxiv.py

```diff
@@ -140,8 +142,8 @@ def _process_result(self, result):
 
     def _get_next(self):
 
-        n_left = self.max_results
-        start = 0
+        n_left = self.max_results - self.start
```

Comment:
This line should be unchanged; `max_results` defines the maximum number of objects to be delivered by the arXiv API, regardless of the `start` *offset.*

For example, this should return 10 results:

```python
arxiv.query(query_string="quantum", start=2000, max_results=10)
```